content/deployment/install.md

```diff
@@ -209,304 +147,85 @@ Once you downloaded and verified the release, you are ready to continue with the
-`/etc/apache2/`, or `/usr/local/apache2/` depending on the envirovment).
-Typically we reccomend following the Fedora practice of creating a
-folder specificlly for ModSecurity rules. In our example we have named
-this modsecurity.d and placed in within the root Apache directory. When
-using Apache we can use the wildcard notation to vastly simplify our
-rules. Simply copying our cloned directory to our modsecurity.d folder
-and specifying the appropertie include directives will allow us to
-install OWASP CRS. In the example below we have also included our
-`modsecurity.conf` file which includes reccomended configurations for
-ModSecurity
+We recommend you creating a folder specifically for the rules. In our example we have named this `modsecurity.d` and placed in within the root `<web server config>/` directory. When using Apache we can use the wildcard notation to vastly simplify our rules. Simply copying our cloned directory to our `modsecurity.d` folder and specifying the appropiate include directives will allow us to install OWASP CRS. In the example below we have also included our `modsecurity.conf` file which includes reccomended configurations for ModSecurity
 
 ```apache
 <IfModule security2_module>
         Include modsecurity.d/modsecurity.conf
-        Include modsecurity.d/owasp-modsecurity-crs/csr-setup.conf
-        Include modsecurity.d/owsp-modsecurity-crs/rules/*.conf
+        Include modsecurity.d/coreruleset-{{< param crs_latest_release >}}/csr-setup.conf
+        Include modsecurity.d/coreruleset-{{< param crs_latest_release >}}/rules/*.conf
 </IfModule>
 ```
 
 ### Includes for Nginx
 
-Nginx will include from the Nginx conf directory (`/usr/local/nginx/conf/`
-depending on the envirovment). Because only one `ModSecurityConfig`
-directive can be specified within nginx.conf we reccomend naming that
-file `modsec_includes.conf` and including additional files from there. In
-the example below we copied our cloned `owasp-modsecurity-crs` folder into
-our Nginx configuration directory. From there we specifying the
-appropertie include directives which will include OWASP CRS when the
-server is restarted. In the example below we have also included our
-`modsecurity.conf` file which includes reccomended configurations for
-ModSecurity
+Nginx will include from the Nginx conf directory (`/etc/nginx` or `/usr/local/nginx/conf/` depending on the environment). Because only one `ModSecurityConfig` directive can be specified within nginx.conf we recommend naming that file `modsec_includes.conf` and including additional files from there. In the example below we copied our cloned `owasp-modsecurity-crs` folder into our Nginx configuration directory. From there we specify the appropiate include directives which will include OWASP CRS when the server is restarted. In the example below we have also included our `modsecurity.conf` file which includes reccomended configurations for ModSecurity
```

Comment:
```suggestion
Nginx will include from the Nginx conf directory (`/etc/nginx` or `/usr/local/nginx/conf/` depending on the environment). Because only one `ModSecurityConfig` directive can be specified within `nginx.conf` we recommend naming that file `modsec_includes.conf` and including additional files from there. In the example below, we copied our cloned `owasp-modsecurity-crs` directory into our Nginx configuration directory. From there, we specify the appropriate include directives which will include OWASP CRS when the server is restarted. In the example below, we have also included our `modsecurity.conf` file which includes recomended configurations for ModSecurity:
```